src/main.ts

```diff
@@ -2,12 +2,12 @@ import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
 
 async function start() {
-  const PORT = process.env.POST || 3333;
+  const PORT = process.env.PORT || 8080;
```

Comment:
Do not specify default port. It can break microservices structure in case of port duplications. Better to add own ConfigModule with .env validations.